README.md

```diff
@@ -33,67 +33,58 @@ The Linera protocol repository is broken down into the following crates and subd
 
 10. [`examples`](./examples) Examples of Linera applications written in Rust.
 
-## Quickstart with the Linera service CLI
+Additionally,
+- `linera-service-graphql-client` is a Rust GraphQL client for the system components of the `linera-service`
+- `linera-explorer` contains an experimental block explorer.
+- `linera-indexer` is an experimental block indexer.
+- `linera-witty` is an experimental alternative to `wit-bindgen` for rust hosts.
 
-The following script can be run with `cargo test`.
+## Quickstart
 
-```bash
-storage="ROCKSDB"
+See our [developer manual](https://linera.dev) for a full introduction to the Linera SDK.
+
+The following script creates a local test network and runs a simple transfers between two user chains.
+
+It can be run with `cargo test readme`.
 
+```bash
 if [ $# -eq 1 ]
 then
-    if [ "$1" == 'DYNAMODB' ]
-    then
-        storage="DYNAMODB"
-    fi
-    if [ "$1" == 'SCYLLADB' ]
-    then
-        storage="SCYLLADB"
-    fi
+    STORAGE="$1"
+else
+    STORAGE="ROCKSDB"
```

Comment:
I'd rather not support databases other than RocksDB in the README example, and explain those in the manual instead. It would make the README a lot more readable.